core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -1282,20 +1282,19 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     )
   }
 
-  // TODO: This test will need to be deleted when implementing https://broadworkbench.atlassian.net/browse/CA-947
```

Comment:
This TODO is still relevant I believe -- once we remove the Terra billing project creation status entirely, we won't be able to return a 400 if it's not Ready

Reply:
How else can you tell if a v1 billing project failed to be created? There not separate classes?

Reply:
For posterity, CA-947 involves deleting all errored billing projects, therebye making this change obsolete on completion.

Reply:
There isn't another way to tell that comes to mind. It is possible that we will decide not to to CA-947 at all in favor of keeping the creation status (even if it's a bit obsolete for v2 projects), but I'm not sure. We can remove this TODO if we close out CA-947